utils/Sounds.ts

```diff
@@ -0,0 +1,49 @@
+import { Audio } from "expo-av";
+
+enum Sounds {
+  NEXT = require("app/assets/sounds/nextbeep.mp3"),
+  FIRST = require("app/assets/sounds/firstbeep.mp3"),
+}
+
+const firstSound = new Audio.Sound();
+const nextSound = new Audio.Sound();
+
+async function loadSound() {
+  // Loading Sound
+  try {
+    await firstSound.loadAsync(Sounds.FIRST);
+    await nextSound.loadAsync(Sounds.NEXT);
+  } catch (error) {
+    console.log(error);
+  }
+}
+
+loadSound();
+
+async function playFirstSound() {
+  // Playing Sound
+  if (firstSound) {
+    await firstSound.setPositionAsync(0);
+    await firstSound.playAsync();
+  }
+}
+
+async function playNextSound() {
+  // Playing Sound
+  if (nextSound) {
+    await nextSound.setPositionAsync(0);
+    await nextSound.playAsync();
+  }
+}
```

Comment:
Since they are doing the same logic with different `Audio` objects. I recommend you merge them and use them as an argument so that you have to write less changes in the future

Reply:
Isn't `loadSound()` called when this file is generated? If so you don't have to do logic for initialization of `Sound` object within this function. If not then you can merge them and set if statement to check what to call depending on an argument. It's better to have one function for the sake of maintainability.

```suggestion
}

type soundNameType = "first" | "second"

async function playSoundFromStart(soundName: soundNameType) { 
    let sound = soundName === 'first' ?  firstSound : secondSound
    await sound.setPositionAsync(0);
    await sound.playAsync();
}
```

Reply:
Please Check the latest iteration I've done! Thanks.